### ISSUE TITLE
rpc Report(ReportRequest) returns (ReportResponse) 정의

### ISSUE THREAD

**Issue by kkweon** [codingpot/pr12er#89]
```proto
message ReportRequest {
  enum ReportType {
    REPORT_UNSPECIFIED = 0;
    REPORT_NEW_PR_VIDEO = 1;
    REPORT_BUG = 2;
  }
  ReportType type = 1;
  string message = 2;
}
```

**Comment by kkweon** [comment]
https://www.figma.com/proto/6furApgnPQzJ1vnLUY1kdm/pr12er-report-mock?node-id=1%3A2&scaling=scale-down&page-id=0%3A1

**Comment by kkweon** [comment]
implemented in #152
